### DISCUSSION TITLE
Strange log output

### DISCUSSION THREAD

**Discussion by nowost** [apache/airflow#30120] [Q&A]
Hello, please help! 
I have two airflow clusters. All config files, settings, OS, python libs are identical. I run the same DAG, but in one cluster i got extended CONTEXT: message in log output and no CONTEXT: message in log  at second cluster.
I trying to find, where i can  disable CONTEXT: message in log , because log is unredable.

**Comment by potiuk** [comment]
If you won't provide examples of the messages including stacktraces / lines weehre they are coming from, airflow version and more details, I doubt anyone will be able to help you, because no-one knows what you re talking about.
